catalog/pom.xml

```diff
@@ -4127,7 +4127,7 @@
                         <!-- prepare the catalog and update doc files, etc. -->
                         <goals>
                             <goal>prepare-catalog-quarkus</goal>
-                            <goal>update-doc-extensions-list</goal>
+                            <!-- <goal>update-doc-extensions-list</goal> -->
```

Comment:
Can this be removed?

Reply:
Yes, I want to remove the whole mojo - see my commet above

> Remove UpdateDocExtensionsListMojo, its templates and config